### PR TITLE
Update rubygem-smart_proxy_ansible to 3.2.1

### PR DESCRIPTION
**Pull request by adamruzicka**

### CODE REVIEW

**Comment by adamruzicka** [comment]
CC @ezr-ondrej just to double check

File: packages/plugins/rubygem-smart_proxy_ansible/rubygem-smart_proxy_ansible.spec [hunk 1]
```diff
@@ -28,16 +28,18 @@ Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
 Requires: ansible >= 2.2
 
 Requires: %{?scl_prefix}rubygem(smart_proxy_dynflow_core) >= 0.1.5
```
Comment (ezr-ondrej) [inline packages/plugins/rubygem-smart_proxy_ansible/rubygem-smart_proxy_ansible.spec:30]: Do we want to keep this?

**Comment by adamruzicka** [comment]
Let's hold this a bit, 3.2.1 is coming in a moment

**Comment by ezr-ondrej** [comment]
It seems we need to include the bin/json... to `%files`
```
error: Installed (but unpackaged) file(s) found:
   /opt/theforeman/tfm/root/usr/share/gems/gems/smart_proxy_ansible-3.2.1/bin/json_inventory.sh
    Installed (but unpackaged) file(s) found:
   /opt/theforeman/tfm/root/usr/share/gems/gems/smart_proxy_ansible-3.2.1/bin/json_inventory.sh
```

**Review by ezr-ondrej — APPROVED** [review]
The PR title is now bit missleading as we upgrade to 3.2.1 and I believe it's time to squash.
Apart of it LGTM :+1:

**Review by ehelms — APPROVED** [review]

**Pull request merged**